src/app/home/home.dashboard.ts

```diff
@@ -215,35 +215,67 @@ export const homeDashboard: DashboardDefaultConfiguration = {
                 title: 'Latency > 1s',
                 subscript: '%',
                 data: {
-                  type: 'calls-percentage-data-source',
-                  context: 'API_TRACE',
-                  'call-count-metric-key': 'calls',
-                  filters: [
-                    {
-                      type: 'graphql-key-value-filter',
-                      key: 'duration',
-                      operator: GraphQlOperatorType.GreaterThan,
-                      value: 1000
+                  type: 'percentage-data-source',
```

Comment:
data source that depends on other data sources. Should we rename it differently? `percentage-composite-data-source` may be?

Reply:
Makes sense. Will merge this and send a follow up PR modifying the name.